### ISSUE TITLE
--help-examples option fails in CLI version 7.10.2

### ISSUE THREAD

**Issue by adam-wolfe** [zowe/zowe-cli#1617]
**Describe the bug**

--help-examples does not appear to be working in version 7.10.2. When the --help-examples option is specified, a command error is displayed.

**Expected and actual results**

Details about the behavior:
1. Command issued: zowe profiles create --help-examples
2. Expected results: List of example commands
3. Actual results: 

```
Command Error:
Unknown arguments: help-examples, helpExamples
Command failed due to improper syntax
Did you mean: profiles create base-profile?

Command entered: "profiles create --help-examples"
Available commands are "zosmf-profile, tso-profile, ssh-profile, endevor-profile, endevor-location-profile, zftp-profile, db2-profile, cics-profile, ims-profile, mq-profile, base-profile".
Use "zowe profiles create --help" to view groups, commands, and options.
Error Details:
Unknown arguments: help-examples, helpExamples
```

**Describe your environment**

- Zowe CLI version installed: 7.10.2

Gif showing behavior in 7.10.2 and 7.10.1:
![help-examples-issue2](https://user-images.githubusercontent.com/122040687/216170354-a81e3ba2-58bb-481f-8294-22af4217d345.gif)

**Comment by abwolfe** [comment]
Same behavior confirmed on Mac w/ Zowe CLI 7.10.2.

**Comment by t1m0thyj** [comment]
Looks like GitHub closed the issue prematurely.

To complete the fix, we need to get Imperative 5.8.3 published, and update Imperative in Zowe CLI.

**Comment by zFernand0** [comment]
We only need to update the CLI now (https://github.com/zowe/imperative/pull/935#issuecomment-1419217427)
![image](https://user-images.githubusercontent.com/37381190/217006604-d3d37629-7f00-4134-9858-3be630b40582.png)

**Comment by zFernand0** [comment]
Should we close this, now that the CLI was updated with the imperative version mentioned above?
https://github.com/zowe/zowe-cli/pull/1606#issuecomment-1420851256

**Comment by adam-wolfe** [comment]
This issue is resolved in Zowe CLI 7.10.4.
